examples/robot_example.cpp

```diff
@@ -70,6 +70,7 @@ int main(int argc, char **argv)
     // Define robot options
     kodlab::mjbots::ControlLoopOptions options;
     options.log_channel_name = "motor_data";
+    options.input_channel_name = "mode_input";
```

Comment:
Did we never fix this oversight? or this a default?

Reply:
Haha nope never fixed it, and I run into it every time I try running new input tests!